### PR TITLE
feat: auto grab cookies from the user's Chrome or Edge

### PR DESCRIPTION
**Pull request by ysmood**
The only requirement to use this feature is to close Chrome or Edge once ([why](https://github.com/go-rod/rod/blob/70da96e1b0ead2e656c99095c0969682db77f542/lib/examples/use-rod-like-chrome-extension/main.go#L21)), doesn't support other browsers yet.

This feature is completely transparent to users, when it doesn't work annie won't be affected by it at all.

As you can see, I cd to the root of the repo and run without the "-c" flag, I can still get the 4K video:

```bash
$ go run . -i https://www.bilibili.com/video/BV1fK4y1t7hj
Warning: Multi thread download is no longer supported by BiliBili, use single thread instead.

 Site:      哔哩哔哩 bilibili.com
 Title:     活动作品【4K 120FPS】你的设备顶得住吗？4K120帧技术演示片
 Type:      video
 Streams:   # All available quality
     [120]  -------------------
     Quality:         超清 4K
     Size:            494.55 MiB (518571288 Bytes)
     # download with: annie -f 120 ...

...
```

### CODE REVIEW

**Comment by ysmood** [comment]
Also, rod can be used for other scenarios, not just cookies. When a website is hard to parse or scrape we can use rod to simply reverse-engineering work.

File: cookier/cookier.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-rod/rod"
+	"github.com/go-rod/rod/lib/devices"
+	"github.com/go-rod/rod/lib/launcher"
+)
+
+// Get cookies from the user's Chrome or Edge automatically.
```
Comment (iawia002) [inline cookier/cookier.go:12]:
```suggestion
// Get gets cookies from the user's Chrome or Edge automatically.
```

File: cookier/cookier.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	if pages.Empty() {
+		page = browser.MustPage("")
+		defer page.MustClose()
+	} else {
+		page = pages.First()
+	}
+
+	cookies := page.MustCookies(urls...)
+
+	list := []string{}
```
Comment (iawia002) [inline cookier/cookier.go:29]:
```suggestion
	list := make([]string, 0, len(cookies))
```

**Review by iawia002 — COMMENTED** [review]
Thanks, basically LGTM, but I got an error when using this feature:

```
fork/exec /var/folders/wd/rkkl_chj2fg52z_qqts9r82c0000gp/T/rod/chromium-809590/chrome-mac/Chromium.app/Contents/MacOS/Chromium: no such file or directory
```

**Comment by ysmood** [comment]
> Thanks, basically LGTM, but I got an error when using this feature:

@iawia002 It works fine to me, can you give more details?

**Comment by ysmood** [comment]
> but I got an error when using this feature: 

@iawia002 I fixed issue, can you try it again?

**Comment by iawia002** [comment]
Thanks for the quick fix, it works, and can you also update the documentation at https://github.com/iawia002/annie#cookies? Introducing this feature and the requirements

**Comment by ysmood** [comment]
Updated the readme on #820
